app/models/relationship.rb

```diff
@@ -0,0 +1,49 @@
+class Relationship < ApplicationRecord
```

Comment:
Are we going to have relationships also show up on the audit log? Or is that unneccessary.